### DISCUSSION TITLE
Does V8 support flex-based tables?

### DISCUSSION THREAD

**Discussion by sebpowell** [TanStack/table#4977] [Q&A] [answered]
I've used React Table in the past, and have recently started a new project using V8. 

All the examples I have seen so far use regular table tags. In the past, I've used the `useFlexLayout` plugin, but I can't see any mention of it in the latest version of the docs, and I haven't been able to find any examples of tables being created using this approach. I'm assuming this plugin is deprecated?

Any guidance would be greatly appreciated, thank you in advance.

**Comment by Zertz** [comment, marked as answer]
The reason `useFlexLayout` no longer exists is that it's not needed anymore.

I removed the hook from `useTable` and updated the root node for cells:

react-table@7:
```tsx
<div {...cell.getCellProps({ className: "..." })}>
```

react-table@8:
```tsx
<div key={cell.id} className="..." style={{ width: cell.column.getSize() }}>
```

**Reply by User6531** [reply]
@Zertz  Could you please share a full working example?

**Reply by verylovestars** [reply]
> @Zertz Could you please share a full working example?

Same question

**Reply by KevinVandy** [reply]
Lots of our examples use flexbox or grid styles, like the virtualized or some of the column resizing examples. Just use the CSS how you want to
